Decompiler/Program.cs

```diff
@@ -115,6 +129,11 @@ private static void ProcessFile(string path)
             {
                 ParseVPK(path);
 
+                if (File.Exists(path + ".manifest.txt"))
+                {
+                    File.Delete(path + ".manifest.txt");
```

Comment:
Why are you deleting the manifest here? Deleting it like this renders it useless.